main_app/pubspec.yaml

```diff
@@ -41,6 +41,8 @@ dev_dependencies:
 
 # The following section is specific to Flutter.
 flutter:
+  assets:
+    - ../design_system/assets/
```

Comment:
This allows the assets on the design_system to be used directly inside the main_app. Ideally the assets should be inside the lib folder, so we can use the specific syntax here with `packages/design_system/...`